CMakeLists.txt

```diff
@@ -37,6 +37,10 @@ if(NOT CRASHREPORTER_EXECUTABLE)
     set(CRASHREPORTER_EXECUTABLE "${APPLICATION_EXECUTABLE}_crash_reporter")
 endif()
 
+if(DEFINED APPLICATION_SERVER_URL)
```

Comment:
Do we really need that hunk? I mean all the other branding variables we just set and don't turn them into options on the command line.

Reply:
Not required anymore due to the change to `set` in `NEXTCLOUD.cmake`.